ReactAndroid/src/main/java/com/facebook/react/bridge/CatalystInstanceImpl.java

```diff
@@ -297,6 +299,9 @@ public void run() {
       }
     });
 
+    MessageQueueThreadRegistry.clear();
```

Comment:
What MessageQueueThread is associated with the UI thread? I don't think this does anything.